DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/experimental/platform_designer/add-quartus-sln/.gitignore

```diff
@@ -10,7 +10,7 @@ add.qws
 add_kernel_wrapper
 
 # ignore oneAPI generated IP
-add.fpga_ip_export.prj_1
+*.prj
```

Comment:
Please remove this .gitignore file from the PR

Reply:
done